stubs/networkx/@tests/stubtest_allowlist.txt

```diff
@@ -0,0 +1,21 @@
+# overloading a class-decorated method crashes stubtest
```

Comment:
https://github.com/python/mypy/issues/16840
```suggestion
# overloaded class-decorators are not properly supported in type-checkers:
# - mypy: https://github.com/python/mypy/issues/16840
# - pyright: https://github.com/microsoft/pyright/issues/7167
```
Current version doesn't cause a stubtest crash anymore. But I'll still try to replicate the crashes I had in a fork to submit a crash report.